vaadin-select-flow-parent/vaadin-select-flow/src/main/java/com/vaadin/flow/component/select/Select.java

```diff
@@ -185,10 +185,10 @@ public Select(T... items) {
      * are not the correct ones, e.g. the list box is the only child of select,
      * even though that is not visible from the component level.
      */
```

Comment:
This file still mentions "vaadin-item" and "vaadin-list-box". The outdated mention of `"vaadin-select.html imports vaadin-list-box.html."` could be removed.

Reply:
Done.